pkl-core/src/main/java/org/pkl/core/runtime/Identifier.java

```diff
@@ -152,6 +152,9 @@ public final class Identifier implements Comparable<Identifier> {
   // common in lambdas etc
   public static final Identifier IT = get("it");
 
+  // dummy, unrepresentable identifier
+  public static final Identifier DUMMY = get("`#_");
```

Comment:
Let's use this name too in `FunctionNode.java`:

```diff
diff --git a/pkl-core/src/main/java/org/pkl/core/ast/member/FunctionNode.java b/pkl-core/src/main/java/org/pkl/core/ast/member/FunctionNode.java
index 31c7d8791..bdedbda41 100644
--- a/pkl-core/src/main/java/org/pkl/core/ast/member/FunctionNode.java
+++ b/pkl-core/src/main/java/org/pkl/core/ast/member/FunctionNode.java
@@ -165,8 +165,8 @@ public final class FunctionNode extends MemberNode {
     var parameters = CollectionUtils.<String, PType>newLinkedHashMap(paramCount);
     for (var i = 0; i < paramCount; i++) {
       var slotName = getFrameDescriptor().getSlotName(i);
-      // Ignored parameters (`_`) have no name
-      var paramName = slotName == null ? "_#" + i : slotName.toString();
+      // This is a blank identifier; use dummy identifier name (impossible to represent in Pkl code)
+      var paramName = slotName == null ? Identifier.DUMMY.toString() + i : slotName.toString();
       parameters.put(paramName, TypeNode.export(parameterTypeNodes[i]));
     }
 

```

Reply:
Ah, scratch that. These names need to be unique because this is part of the reflect API.